compiler/passes/scopeResolve.cpp

```diff
@@ -1922,8 +1922,8 @@ static void resolveEnumeratedTypes() {
         if (EnumType* type = toEnumType(first->var->type)) {
           if (SymExpr* second = toSymExpr(call->get(2))) {
             const char* name;
-            bool found = false;
-
+            //bool found = false;
+            /*
```

Comment:
even after making the _this change, it still fails to flag paren-less functions as calls. 

Since a paren-less function appears to be indistinguishable at this stage from an invalid enum symbol, I chose to let it pass through to function resolution, where it is caught with an "unresolved enum symbol or call" message.